### PR TITLE
Fixed width of no-data screen tabs and FAQ link update for WordPress image tracking.

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Fixed width of no-data screen tabs and FAQ link update for WordPress image tracking.
Fixes:  #PG-2776, #PG-2790

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/SitesManager/templates/_siteWithoutDataTabs.twig [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 <div class="row">
     <div class="col s12">
-        <ul class="tabs" style="overflow: auto !important;height: auto !important;">
+        <ul class="tabs no-data-screen-ul-tabs" style="overflow: auto !important;height: auto !important;">
```
Comment (sgiehl) [inline plugins/SitesManager/templates/_siteWithoutDataTabs.twig]: would you mind moving the css in the style property to the less file as well?

Comment (AltamashShaikh) [reply plugins/SitesManager/templates/_siteWithoutDataTabs.twig]: updated

**Comment by sgiehl** [comment]
@AltamashShaikh The fixed width of the tabs have been removed on purpose, right? Couldn't find any acceptance criteria for that in the referenced issues...

**Comment by AltamashShaikh** [comment]
> @AltamashShaikh The fixed width of the tabs have been removed on purpose, right? Couldn't find any acceptance criteria for that in the referenced issues...

@sgiehl The correct issue is `PG-2776` updated the description, sorry was mistakenly updated a wrong issue no

**Review by sgiehl — APPROVED** [review]

**Pull request merged**